src/ext.js

```diff
@@ -126,16 +142,98 @@ export default function ext(/* env */) {
+                    {
+                      value: false,
+                      translation: 'properties.off',
+                    },
+                  ],
+                },
+                backgroundUrl: {
+                  ref: 'style.background.url.qStaticContentUrlDef.qUrl',
+                  layoutRef: 'style.background.url.qStaticContentUrl.qUrl',
+                  schemaIgnore: true,
+                  translation: 'Common.Image',
+                  tooltip: { select: 'properties.media.select', remove: 'properties.media.removeBackground' },
+                  type: 'string',
+                  component: 'media',
+                  defaultValue: '',
+                  show(data) {
+                    return getValue(data, 'style.background.isUsed');
+                  },
+                },
+                backgroundSize: {
+                  ref: 'style.background.size',
+                  translation: 'properties.backgroundImage.size',
+                  type: 'string',
+                  component: 'dropdown',
+                  defaultValue: 'auto',
+                  options: [
+                    {
+                      value: 'auto',
+                      translation: 'properties.backgroundImage.originalSize',
+                    },
+                    {
+                      value: 'alwaysFit',
+                      translation: 'properties.backgroundImage.sizeAlwaysFit',
+                    },
+                    {
+                      value: 'fitWidth',
+                      translation: 'properties.backgroundImage.sizeFitWidth',
+                    },
+                    {
+                      value: 'fitHeight',
+                      translation: 'properties.backgroundImage.sizeFitHeight',
+                    },
+                    {
+                      value: 'fill',
+                      translation: 'properties.backgroundImage.sizeStretch',
+                    },
+                  ],
+                  show(data) {
+                    return (
+                      getValue(data, 'style.background.isUsed')
+                      && getValue(data, 'style.background.url.qStaticContentUrlDef.qUrl')
+                    );
+                  },
+                },
+                backgroundPosition: {
+                  ref: 'style.background.position',
+                  translation: 'Common.Position',
+                  type: 'string',
+                  component: 'align-matrix',
+                  defaultValue: 'topLeft',
+                  show(data) {
+                    return (
+                      getValue(data, 'style.background.isUsed')
+                      && getValue(data, 'style.background.url.qStaticContentUrlDef.qUrl')
+                      && getValue(data, 'style.background.size') !== 'fill'
+                    );
+                  },
+                  currentSize(data) {
+                    return getValue(data, 'style.background.size');
+                  },
```

Comment:
What does this one do?

Reply:
i think this is what align-matrix uses to watch the changes in size?